src/youtube-player/youtube-player.ts

```diff
@@ -135,10 +135,10 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
 
   /** Width of video player */
   @Input()
-  get width(): number | undefined {
+  get width(): number | string | undefined {
```

Comment:
If we're making this change to `width` shouldn't we make it for `height` as well?

Reply:
@mmalerba Done